RetailCoder.VBE/Inspections/InspectionsUI.resx

```diff
@@ -304,7 +304,7 @@
     <value>Variable is used but not assigned</value>
   </data>
   <data name="UntypedFunctionUsageInspectionMeta" xml:space="preserve">
-    <value>A string-returning equivalent function exists and should preferably be used to avoid implicit type conversions.</value>
+    <value>A string-returning equivalent function exists and should preferably be used to avoid implicit type conversions. However, this function will crash if passed a 'Null' value, so do not use it if the value passed can be 'Null'.</value>
```

Comment:
Nice.  I didn't like that wording too much myself, actually, but figured I'd get the ball rolling.